DigitalLearningSolutions.Web/ViewModels/Login/ChooseACentreViewModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Login
+{
+    using System.Collections.Generic;
+
+    public class ChooseACentreViewModel
+    {
+        public ChooseACentreViewModel()
+        {
+            CentreUserDetails = new List<CentreUserDetails>();
```

Comment:
Would it be better to set the userDetails in the constructor rather than declaring it as an empty list and then setting the field?